enos/modules/verify_secrets_engines/modules/create/auth.tf

```diff
@@ -143,3 +145,84 @@ resource "enos_remote_exec" "auth_login_testuser" {
     }
   }
 }
+
+# Enable ldap auth
+resource "enos_remote_exec" "auth_enable_ldap" {
+  environment = {
+    AUTH_METHOD       = "ldap"
+    AUTH_PATH         = local.auth_ldap_path
+    VAULT_ADDR        = var.vault_addr
+    VAULT_TOKEN       = var.vault_root_token
+    VAULT_INSTALL_DIR = var.vault_install_dir
+  }
+
+  scripts = [abspath("${path.module}/../../scripts/auth-enable.sh")]
+
+  transport = {
+    ssh = {
+      host = var.leader_host.public_ip
+    }
+  }
+}
+
+# Write the ldap config
+resource "enos_remote_exec" "auth_write_ldap_config" {
+  depends_on = [
+    enos_remote_exec.auth_enable_ldap
+  ]
+
+  environment = {
+    AUTH_PATH         = local.auth_ldap_path
+    GROUPATTR         = "memberOf"
+    GROUPDN           = "CN=Users,DC=corp,DC=example,DC=net"
+    INSECURE_TLS      = "true"
+    POLICIES          = local.auth_ldap_path
+    UPNDOMAIN         = "corp.example.net"
+    URL               = "ldaps://ldap.example.com"
+    USERATTR          = "sAMAccountName"
+    USERDN            = "CN=Users,DC=corp,DC=example,DC=net"
+    VAULT_ADDR        = var.vault_addr
+    VAULT_INSTALL_DIR = var.vault_install_dir
+    VAULT_TOKEN       = var.vault_root_token
+  }
+
+  scripts = [abspath("${path.module}/../../scripts/auth-ldap-write.sh")]
+
+  transport = {
+    ssh = {
+      host = var.leader_host.public_ip
+    }
+  }
+}
+
+# Update the ldap config
```

Comment:
Can we leave a note/comment about why we're updating it? I feel like there's some implied context here that will be easy to miss later on.